Document Explain API search_pipeline limitation (#12096)
opensearch-project/documentation-website: _vector-search/ai-search/hybrid-search/explain.md

To use the `explain` parameter, you must configure the `hybrid_score_explanation` response processor in your search pipeline. For more information, see [Hybrid score explanation processor]({{site.url}}{{site.baseurl}}/search-plugins/search-pipelines/explanation-processor/). 

### Explain by document ID

You can use the [Explain API]({{site.url}}{{site.baseurl}}/api-reference/explain/) with a hybrid query to get scoring details for a single document:

```json
GET <index>/_explain/<id>
POST <index>/_explain/<id>
```

In this case, the result contains only raw Lucene-level scoring information, for example, [Okapi BM25](https://en.wikipedia.org/wiki/Okapi_BM25) scores for text-based subqueries such as `term` or `match`. For an example response, see [Explain API example response]({{site.url}}{{site.baseurl}}/api-reference/explain/#example-response).

The Explain API has the following limitations when used with hybrid queries:

> - **No search pipeline support**: The `_explain` endpoint does not support the `search_pipeline` parameter, either inline or as a URL parameter. Sending a request with `search_pipeline` results in a `parsing_exception` error. This limitation applies to all query types, not only hybrid queries.

Reviewer: ```suggestion
- The `_explain` endpoint does not support the `search_pipeline` query parameter, either inline or as a query parameter. Providing a `search_pipeline` parameter results in a `parsing_exception` error. This limitation applies to all query types, not only hybrid queries.
```
